pages.pt_BR/linux/htop.md

```diff
@@ -0,0 +1,15 @@
+# htop
+
+> Visualizar informação dinâmica em tempo real acerca de processos em execução. Uma versão melhorada do comando `top`.
+
+- Inicializar htop:
+
+`htop`
+
+- Inicializar htop mostrando somente processos pertencentes a um usuário:
+
+`htop -u {{nome_usuário}}`
+
+- Obter ajuda acerca de comandos interativos:
```

Comment:
```suggestion
- Obter ajuda acerca de comandos interativos, este comando deve ser usando com o htop em execução :
```

Reply:
Se usarmos parêntesis evitamos um [comma splice](https://en.wikipedia.org/wiki/Comma_splice):

```suggestion
- Obter ajuda acerca de comandos interativos (este comando deve ser usando com o htop em execução):
```

Reply:
Again, I would not advise changing the original phrase in the translation. This change should be done in a different PR IMHO.

Reply:
While I disagree in principle with avoiding improvements in translation PRs, I do agree tha the page is quite workable without the suggested edits, so I would have no objection to merging it without these changes, and doing them separately. 👍